goto/commands/default.py

```diff
@@ -4,18 +4,32 @@
 from .open import open
 
 
-def default(magic, magicword):
+def default(magic, command, args):
```

Comment:
command should be `None` in this case. 
It is a bit confusing to concatenate args and command.

Reply:
Ahh, of course. On it